glados.py

```diff
@@ -103,8 +104,18 @@ def __init__(
         self.wake_word = wake_word
         self._vad_model = vad.VAD(model_path=str(Path.cwd() / "models" / VAD_MODEL))
         self._asr_model = asr.ASR(model=str(Path.cwd() / "models" / ASR_MODEL))
-        self._tts = tts.Synthesizer(model_path=str(Path.cwd() / "models" / VOICE_MODEL), use_cuda=False)
-
+        if shutil.which("espeak-ng"):
+            espeak = "espeak-ng"
+        elif shutil.which("espeak"):
+            espeak = "espeak"
+        else:
+            logger.error("I cannot find `espeak` on your path. You probably "
+                                    "forgot to install it. How utterly embarrassing "
+                                    "for you.")
+            raise FileNotFoundError("Cannot find espeak")
+        self._tts = tts.Synthesizer(model_path=str(Path.cwd() / "models" / VOICE_MODEL),
+                                     use_cuda=False, espeak=espeak)
```

Comment:
Instead of adding more and more code to the core, I think it would be better if the espeak-detection was within the wrapper itself (with an optional parameter for manual overrides).